src/main/java/com/github/thedeathlycow/moregeodes/features/ModConfiguredFeatures.java

```diff
@@ -213,6 +213,35 @@ public class ModConfiguredFeatures {
                         0.3f
                 )
         ));
+
+        if (FabricLoaderImpl.INSTANCE.isModLoaded("ae2")) {
+            CERTUS_GEODE = register("certus_geode", new ConfiguredFeature<>(Feature.GEODE, new GeodeFeatureConfig(
+                    new GeodeLayerConfig(
+                            SimpleBlockStateProvider.of(Blocks.WATER.getDefaultState()),
+                            SimpleBlockStateProvider.of(AEBlocks.DAMAGED_BUDDING_QUARTZ.block()),
+                            SimpleBlockStateProvider.of(AEBlocks.QUARTZ_BLOCK.block()),
+                            SimpleBlockStateProvider.of(Blocks.OBSIDIAN),
+                            SimpleBlockStateProvider.of(AEBlocks.SKY_STONE_BLOCK.block()),
+                            List.of(AEBlocks.SMALL_QUARTZ_BUD.block().getDefaultState(),AEBlocks.MEDIUM_QUARTZ_BUD.block().getDefaultState()),
```

Comment:
missing large quartz bud and quartz cluster?

Reply:
Nah. Just felt like if the player wanted the certus so bad, they could afford to be patient. Also, geodes are deep enough to usually have some ticking time before being reached by the players. But they can be added if you wish, tis your mod after all.